### DISCUSSION TITLE
Not working on my HP 24 f monitor

### DISCUSSION THREAD

**Discussion by vmanav** [MonitorControl/MonitorControl#1629] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Read through [the Monitor-Troubleshooting Wiki](https://github.com/MonitorControl/MonitorControl/wiki/Monitor-Troubleshooting)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the issue

Hi, 
I have been using this amazing tool from the last 6 months with my Dell monitor. 

Recently, moved to a new place and changed the monitor to a HP one, but the brightness controls are not working for my HP 24-inch monitor (Monitor Details - [link](https://www.hp.com/id-id/shop/hp-24f-24-inch-display-3al28aa.html))

Can you help with this ?

**Mac Version: Sonoma 14.5**
![Screenshot 2024-09-29 at 9 49 01 AM](https://github.com/user-attachments/assets/075987da-a4ef-4474-8aac-e11638db200f)
![Screenshot 2024-09-29 at 9 52 36 AM](https://github.com/user-attachments/assets/63c76ec2-c061-446a-b0bd-15f2cec38cf3)

### Expected behavior

Monitor controls should work from the navbar and change the brightness of the monitor.

### Environment Information (please complete the following information)

```markdown
- macOS version: Sonoma 14.5
- Mac model: Macbook Air M1
- MonitorControl version: Version 3.1.1 Build 3096
- Monitor(s): HP 24f 24-inch Monitor (link provided in issues description)
- Monitor Cable(s)/Connection(s): 1 HDMI
- Apple Silicon/M1 (yes or no): yes
```

**Comment by waydabber** [comment]
The display or the dongle used might not support DDC control (note: you are using an outdated app version but this should not cause an issue I think).
